fenicsadapter/fenicsadapter.py

```diff
@@ -268,9 +212,9 @@ def write_data(self, write_function):
         else:
             raise Exception("write_function provided is neither VECTOR nor SCALAR type")
 
-    def initialize(self, coupling_subdomain, mesh, function_space, dimensions=2):
+    def initialize(self, coupling_subdomain, mesh, function_space, dimensions=2, write_function=None, fixed_boundary=None):
```

Comment:
dimension should go into constructor

Reply:
we should also make sure to raise an Exception, if `dimension != 2` as long as #1 is not resolved.